### PR TITLE
Add matrix-digest tool

### PR DESCRIPTION
**Pull request by psaavedra**
DEPENDS: https://github.com/psaavedra/matrix-bot/pull/54

A convenient tool to sends the DAYS backlog activity of the ROOM to the
SENDTO mailbox. Ideal to use as a cronjob.

There is an maximum harcoded value of 5000 messages.

Example of use:

```
usage: matrix-digest [-h] [-c CONFFILE] [-s SENDTO] [-d DAYS] ROOM

positional arguments:
  ROOM                  Room

optional arguments:
  -h, --help            show this help message and exit
  -c CONFFILE, --conffile CONFFILE
                        Conffile (default: .matrixbot.cfg)
  -s SENDTO, --sendto SENDTO
                        Send to (default: no-reply@example.domain)
  -d DAYS, --days DAYS  Days (default: 1)
```

Signed-off-by: Pablo Saavedra <psaavedra@igalia.com>

### CODE REVIEW

File: tools/matrix-diggest [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+conffile = "./matrixbot.cfg"
+days = 1
+sendto = "no-reply@example.domain"
+
+## command line options parser #################################################
+parser = argparse.ArgumentParser()
+parser.add_argument("-c", "--conffile", dest="conffile", default=conffile,
+                    help="Conffile (default: %s)" % conffile)
+parser.add_argument("-s", "--sendto", dest="sendto", default=sendto,
+                    help="Sent to (default: %s)" % sendto)
```
Comment (jmallach) [inline tools/matrix-diggest:26]: I assume you want "Send to (default: %s)" here.

**Comment by jmallach** [comment]
The tool should be renamed to `matrix-digest`, along with any other reference to `diggest`, as it's the correct spelling.

File: tools/matrix-diggest (hunk 1 above)
Comment (psaavedra) [reply tools/matrix-diggest]: Fixed. Thanks for spotlighting this.

**Comment by psaavedra** [comment]
> The tool should be renamed to `matrix-digest`, along with any other reference to `diggest`, as it's the correct spelling.

ditto.

**Review by jmunhoz — APPROVED** [review]
LGTM

**Pull request merged**